lib/Runtime/Base/FunctionBody.cpp

```diff
@@ -336,13 +336,13 @@ namespace Js
     bool
     FunctionBody::SkipAutoProfileForCoroutine() const
     {
-        return this->IsCoroutine() && CONFIG_ISENABLED(Js::JitES6GeneratorsFlag);
```

Comment:
Instead of deleting the checks here, shouldn't we instead change the `JitES6Generators` flag in ConfigFlagsList.h to true? This way we can re-disable the feature later if needed.

In ConfigFlagsList.h, change this line:
```c++
FLAGNR(Boolean, JitES6Generators        , "Enable JITing of ES6 generators", DEFAULT_CONFIG_JitES6Generators)
```
And add the default config:
```c++
#define DEFAULT_CONFIG_JitES6Generators (true)
```